Packs/ThalesCipherTrustManager/Integrations/ThalesCipherTrustManager/ThalesCipherTrustManager.yml

```diff
@@ -0,0 +1,2388 @@
+      description: Filter by the user’s username
+      required: false
+      secret: false
+      default: false
+    - name: email
+      isArray: false
+      description: Filter by the user’s email
+      required: false
+      secret: false
+      default: false
+    - name: groups
+      isArray: true
+      description: Filter by users in the given group name. Provide multiple groups separated by comma (',') to get users in all of those groups. Using 'nil' as the group name will return users that are not part of any group.
+      required: false
+      secret: false
+      default: false
+    - name: exclude_groups
+      isArray: true
+      description: Users associated with given group will be excluded from the result. Provide multiple groups separated by comma (',') to exclude multiple groups in the result.
+      required: false
+      secret: false
+      default: false
+    - name: auth_domain_name
+      isArray: false
+      description: Filter by user’s auth domain
+      required: false
+      secret: false
+      default: false
+    - name: account_expired
+      isArray: false
+      description: Filters the list of users whose expiration time has passed.
+      required: false
+      secret: false
+      default: false
+      predefined:
+      - 'true'
+      - 'false'
+      auto: PREDEFINED
+    - name: allowed_auth_methods
+      isArray: true
+      description: Filter by the login authentication method allowed to the users. It is a comma seperated list of values. A special value `empty` can be specified to get users to whom no authentication method is allowed.
+      required: false
+      secret: false
+      default: false
+      predefined:
+      - password
+      - user_certificate
+      - password_with_user_certificate
+      - empty
+      auto: PREDEFINED
+    - name: allowed_client_types
+      isArray: true
+      description: Filter by the client types that can authenticate the user. It is a comma separated list of values.
+      required: false
+      secret: false
+      default: false
+      predefined:
+      - unregistered
+      - public
+      - confidential
+      auto: PREDEFINED
+    - name: password_policy
+      isArray: false
+      description: Filter the list of users based on assigned password policy
+      required: false
+      secret: false
+      default: false
+    - name: return_groups
+      isArray: false
+      description: If set to 'true', it returns the group's name in which user is associated along with all users information.
+      required: false
+      secret: false
+      default: false
+      predefined:
+      - 'true'
+      - 'false'
+      auto: PREDEFINED
+    - name: page
+      isArray: false
+      description: Page to return.
+      required: false
+      secret: false
+      default: false
+    - name: page_size
+      isArray: false
+      description: Number of entries per page. Defaults to 2000 (in case only page was provided). Maximum entries per page is 2000
```

Comment:
```suggestion
      description: Number of entries per page. Defaults to 2000 (in case only page was provided). Maximum entries per page is 2000.
```